pandas/_libs/ops_dispatch.pyx

```diff
@@ -61,7 +70,7 @@ REVERSED_NAMES = {
 
 
 def maybe_dispatch_ufunc_to_dunder_op(
-    object self, object ufunc, str method, *inputs, **kwargs
+    object self, object ufunc, str method, *inputs, bint _no_index=True, **kwargs
```

Comment:
`_no_index` is confusing imo. either `_not_index` or ``_is_index` or `_is_index_object` or ``_not_index_object`

when I see "no index", I think of an object without an index.

Reply:
should probably update the docstring anyway, so that would perhaps make things clear anyway.